tests/agent_server/test_agent_profile_conv_start.py

```
@@ -718,12 +722,12 @@ async def test_profile_launch_inherits_the_stored_memory_preference(
             )
         )
 
-        stored = await _start_from_profile(
+        stored, agent = await _start_from_profile(
```

Ruff (0.16.1)

[warning] Unpacked variable `stored` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)